static/app/components/replays/contextIcon.tsx

```diff
@@ -16,20 +15,15 @@ type Props = {
   showVersion?: boolean;
 };
 
-const LazyContextIcon = lazy(
-  () => import('sentry/components/events/contexts/contextIcon')
-);
+// theme.tsx/iconSizes["sm"] + 4, to make up for padding
+const iconSize = '18px';
```

Comment:
i believe `theme.iconSizes.md` is equivalent to `18px`, can we reuse theme here?